content/docs/reference/collections.md

```diff
@@ -99,11 +100,11 @@ The final pattern used would be
 
 ### Basic Syntax
 
-- `?` matches a single character
-- `*` matches any number of any characters (except `/`).
-- `[abc]` matches any one character in the given set
-- `[a-z]` matches a range.
-- `{foo,bar}` matches any whole word in the given set
+* `?` matches a single character
+* `*` matches any number of any characters (except `/`).
```

Comment:
**nitpick**: Remove full-stop for consistency with other dot points
```suggestion
* `*` matches any number of any characters (except `/`)
```

Reply:
✅🙏